sym-aws-go-approvals/serverless.yml

```diff
@@ -0,0 +1,82 @@
+service: sym-aws-go-approvals
+
+custom:
+  stage: ${{opt:stage, self:provider.stage}}
+  symAccountid: ${{opt:symAccountId, '803477428605'}}
+
+provider:
+  name: aws
+  runtime: go1.x
+
+# you can overwrite defaults here
+  endpointType: REGIONAL
+
+# makes serverless variables double braced so as to not conflict with CF
+  variableSyntax: '\${{(((?!\${{).)+?)}}' 
+
+# you can add statements to the Lambda function's IAM Role here
```

Comment:
is this comment relevant still, or leftover from the template?

Reply:
Still useful IMO for people starting with this template